README.md

```diff
@@ -1 +1,2 @@
 # flyway-repair
+## RETIRED
```

Comment:
For the record, I love the word `Retired` so much more 😂 It's more common though to use `Deprecated` and then a note with something  like `No longer maintained`.

Personally though I think we can just delete this repo as it isn't being used at all and won't be going forward

Reply:
@Smittey ,I am not able to delete (prob some lower permissions on my end). Would you mind doing it whenever you have a free moment?

Reply:
Yeah will do